content/doc/book/security/securing-jenkins.adoc

```diff
@@ -0,0 +1,123 @@
+---
+layout: section
+wip: true
+---
+ifdef::backend-html5[]
+:notitle:
+:description:
+:author:
+:email: jenkinsci-users@googlegroups.com
+ifdef::env-github[:imagesdir: ../resources]
+ifndef::env-github[:imagesdir: ../../resources]
+:toc: left
+endif::[]
+
+= Securing Jenkins
+
+In the default configuration of Jenkins 1.x, Jenkins does not perform any
+security checks. This means the ability of Jenkins to launch processes and
+access local files are available to anyone who can access Jenkins web UI and
+some more.
+
+Securing Jenkins has two aspects to it.
+
+* Access control, which ensures users are authenticated when accessing Jenkins
+  and their activities are authorized.
+* Protecting Jenkins against external threats
+
+== Access Control
+
+You should lock down the access to Jenkins UI so that users are authenticated
+and appropriate set of permissions are given to them. This setting is
+controlled mainly by two axes:
+
+* *Security Realm*, which determines users and their passwords, as well as what
+  groups the users belong to.
+* *Authorization Strategy*, which determines who has access to what.
+
+These two axes are orthogonal, and need to be individually configured. For
+example, you might choose to use external LDAP or Active Directory as the
+security realm, and you might choose "everyone full access once logged in" mode
+for authorization strategy. Or you might choose to let Jenkins run its own user
+database, and perform access control based on the permission/user matrix.
+
+
+* https://wiki.jenkins.io/display/JENKINS/Quick+and+Simple+Security[Quick and Simple Security] --- if you are running Jenkins like `java -jar jenkins.war` and only need a very simple setup
```

Comment:
Doesn't seem to apply any longer, since the default since 2.0 is to be secured initially.
```suggestion
```